tests/unit-tests/UnitTests/Extensions/AddressExtensionsTests.cs

```diff
@@ -15,5 +15,39 @@ public void AppointmentLocation_ShouldReturnTitleCaseString_WhenLocationFound(st
 
             Assert.Equal(expectedResult, result);
         }
+
+        [Theory]
+        [InlineData("{{QUESTION:addressReference}}", "addressReference")]
+        [InlineData("{{QUESTION:addressFromQuestionId}}", "addressFromQuestionId")]
```

Comment:
Not sure we need to test this twice like this, once would be enough I think. Maybe change to a Fact